diff --git a/corev_apu/tb/rvfi_tracer.sv b/corev_apu/tb/rvfi_tracer.sv
index 8d77ee4..ce24108 100644
--- a/corev_apu/tb/rvfi_tracer.sv
+++ b/corev_apu/tb/rvfi_tracer.sv
@@ -70,14 +70,16 @@ module rvfi_tracer #(
           end
         end
         $fwrite(f, "\n");
+        /*
         if (rvfi_i[i].insn == 32'h00000073) begin
           $finish(1);
           $finish(1);
         end
+        */
       end else if (rvfi_i[i].trap)
         $fwrite(f, "exception : 0x%h\n", pc64);
     end
-    if (cycles > SIM_FINISH) $finish(1);
+    // if (cycles > SIM_FINISH) $finish(1);
   end
 
   always_ff @(posedge clk_i or negedge rst_ni)
